### ISSUE TITLE
@layer normalize

### ISSUE THREAD

**Issue by jeffscottward** [chakra-ui/panda#1824]
### Description

Implement a CSS @layer for normalize css.

### Problem Statement/Justification

While CSS reset may nuke everything, normalize gives us a sane cross-browser baseline that doesn’t blow out all the pre-existing defaults. This is generally preferred. We need a cascade layer to do this.

### Proposed Solution or API

@layer normalize

### Additional Information

https://necolas.github.io/normalize.css/latest/normalize.css

**Comment by anubra266** [comment]
@jeffscottward Preflights are generally opinionated. If you'd like something different, you can disable [preflight](https://panda-css.com/docs/references/config#preflight) in your config, and apply your normalize styles through [globalCss](https://panda-css.com/docs/references/config#globalcss) in your config

**Comment by anubra266** [comment]
@jeffscottward By opinionated, what I mean is that the styles applied by reset css differs by developer or design system.
As for your first question, full reset is definitely an opinion, that's why reset can be removed by setting `preflight` to false in your config. 
And like I suggested before, adding your normalize styles to globalCSS, puts them in the `base` layer. That way you're not exactly nuking things, it results exactly as you expect.

In summary, an additional `normalize` layer is not necessary for this purpose.

**Comment by jeffscottward** [comment]
Ok that makes but there is still an issue.

You can't simply npm install and include a normalize or reset stylesheet. You must convert it to a style object.

There is obviously a good tool to do this
https://github.com/postcss/postcss-js

But I bring it up because now you're augmenting the config and bridging between source and target. Is this more optimal than providing at least one suggested (but easily dismissible) native solution?

It's may be ok but should be called out or an example provided in docs then. Otherwise anyone who does "have an opinion" has to go searching for which transform tool among the many will work, is maintained, etc

**Comment by anubra266** [comment]
@jeffscottward If I understand your question correctly, you want to use raw css.
You can send css to layers in your global css file this way.

```css
import url("normalize.css") layer (base)
```

**Comment by anubra266** [comment]
To better understand all the requirements you wish for @jeffscottward Could you give a sample of what you hope it'll be like if there was the `normalize` layer.
Where would you plugin your custom normalize?
Would it be raw css, or a style object?
Would it be in the config or css file?

This way I can give you a picture of the equivalent, and add to documentation where neccessary.

**Comment by jeffscottward** [comment]
> @jeffscottward If I understand your question correctly, you want to use raw css.
> 
> You can send css to layers in your global css file this way.
> 
> 
> 
> ```css
> 
> @import url("normalize.css") layer(base)
> 
> ```

Oh I didn't realize you could import to a layer with that syntax.

Huge feature, tiny explainer on mdn.
![image](https://github.com/chakra-ui/panda/assets/1263486/2525a2bc-0ab0-4410-a521-757a9cab5dfc)

What I was after was, how do I go from `npm install normalize.css sanitize.css` which both have a regular old `.css` file, to including them as layers in my project.

The `globalCSS` you suggested according to the docs is a style object, which would have required conversion. This just felt like a weird place to put it besides needing the conversion, and also the preflight option seems more appropriate.

My suggestion is one of these two.

The simple import you suggested.

Or
`{"preflight": ['reset','normalize','sanitize']}`
instead of being only true false.

**Comment by anubra266** [comment]
Well @jeffscottward I'd say preflight is the popular term inclusive of `resets` or `normalize. Even in libraries like Chakra UI, or Tailwind.

As for using existing libraries like normalize.css or sanitize.css, I suppose the import layer syntax is the best option.
For resets in style object, It's really not weird to be in global css 😁  And you don't exactly have to define it bare there, it could come from a preset. E.g. [pform-reset](https://www.ecopanda.dev/presets/pform-reset)
